README.md

```diff
@@ -61,6 +61,7 @@ import ChipInput from 'material-ui-chip-input'
 |onDelete|`func`||Callback function that is called when a new chip was removed (in controlled mode).|
 |onUpdateInput|`func`||Callback function that is called when the input changes.|
 |placeholder|`string`||A placeholder that is displayed if the input has no values.|
+|alwaysShowPlaceholder|`bool`||If true, the placeholder will always appear after inserting new chips.|
```

Comment:
This should be consistent with the other description, so "If true, the placeholder will always be visible.".

Reply:
Done. But the tests are failing and it is not related to my changes =/